packages/invoices/src/OutboundShipment/DetailView/Footer.tsx

```diff
@@ -94,7 +95,7 @@ export const Footer: FC<OutboundDetailFooterProps> = ({ draft, save }) => {
                   sx={{ fontSize: '12px' }}
                   onClick={() => {
                     success('Saved invoice! 🥳 ')();
-                    save(draft);
+                    save();
```

Comment:
nice to `await save()` and toast afterwards, even though it is too quick to notice when using the mock server 😀

Reply:
ah yes, will do!